### PR TITLE
changed file structure to be compatible with render deployment

### PR DESCRIPTION
**Pull request by cgrq**

### CODE REVIEW

**Comment by cgrq** [comment]
Merging to main

File: README.md [hunk 1]
```diff
@@ -1,3 +1,148 @@
-# Project README
+# Flask React Project
```
Comment (rajahwu) [inline README.md:1]: Change titile

File: frontend/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.4",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5",
```
Comment (rajahwu) [inline frontend/package.json:14]: Change react script version

File: frontend/package.json [hunk 3]
```diff
@@ -38,5 +38,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:5000"
-}
\ No newline at end of file
+  "proxy": "http://127.0.0.1:5000"
```
Comment (rajahwu) [inline frontend/package.json:41]: change proxy

**Review by rajahwu — APPROVED** [review]

**Pull request merged**